wfe2/wfe.go

```diff
@@ -981,7 +981,7 @@ func (wfe *WebFrontEndImpl) RevokeCertificate(
 	}
 
 	// Figure out which type of authentication this JWS uses
-	authType, prob := checkJWSAuthType(jws)
+	authType, prob := checkJWSAuthType(jws.Signatures[0].Header)
```

Comment:
This one, in contrast to my comment above, is fine, because we _know_ that this jws was just returned from parseJWSRequest, and that function guarantees that the jws it returns has a Signatures slice of length exactly 1.